### DISCUSSION TITLE
프론트엔드 라이브러리 논의

### DISCUSSION THREAD

**Discussion by threedalpeng** [threedalpeng/WebGL-Guide#25] [General]
어차피 디자인 싹 갈아 엎으려고 하는 이상 다른 것도 싹 갈아 엎는 것도 나쁘지 않을 것 같아 보입니다.

그래서 현재 사용하고 싶은 프론트 프레임워크나 라이브러리가 있다면 말해주면 좋을 것 같습니다.

예) svelte 쓰고 싶다. 그냥 vue 계속 쓰자 등등등

**Comment by kangrnin** [comment]
그냥 vue 쓰는게 나아보입니다.

**Reply by threedalpeng** [reply]
ㅇㅋ 그럼 이대로

**Comment by threedalpeng** [comment]
혹시 스타일 관련 라이브러리는?

**Reply by kangrnin** [reply]
UnoCSS 그대로 ㄱㄱ
